glommio/src/sys/uring.rs

```diff
@@ -777,7 +777,9 @@ impl PollRing {
         let ring = iou::IoUring::new_with_flags(
             size as _,
             iou::SetupFlags::IOPOLL,
-            iou::SetupFeatures::empty(),
+            iou::SetupFeatures::FAST_POLL
```

Comment:
I don't think this does what you think it does.

Read the kernel source. This is the kernel telling us which features are available, not us picking which features we want to use.

Reply:
This sets the following flag when calling `io_uring_enter`:
```
IORING_FEAT_FAST_POLL

If this flag is set, then io_uring supports using an internal poll 
mechanism to drive data/space readiness. This means that requests that 
cannot read or write data to a file no longer need to be punted to an 
async thread for handling, instead they will begin operation when the 
file is ready. This is similar to doing poll + read/write in userspace, 
but eliminates the need to do so. If this flag is set, requests waiting 
on space/data consume a lot less resources doing so as they are not 
blocking a thread. Available since kernel 5.7.
```

Reply:
Read again: If this flag is set, *THEN IO_URING SUPPORTS* . This is the kernel telling you what it supports, not you asking for the feature.

Reply:
oh lolilol

Reply:
That's embarrassing. Will remove!